src/services/ArtifactService.ts

```diff
@@ -21,3 +22,13 @@ export async function fetchArtifacts(envs: string[]) {
   });
   return data;
 }
+
+export async function uploadArtifacts(artifact: Artifact) {
+  const env = 'test'; // Hard coded env(test) for now
```

Comment:
I think better way is to take env as a default param and set it to 'test'
i.e function uploadArtifacts(artifact: Artifact, env = 'test')